content/influxdb/cloud-dedicated/reference/influxql/functions/misc.md

```diff
@@ -0,0 +1,245 @@
+---
+title: InfluxQL miscellaneous functions
+list_title: Miscellaneous functions
+description: >
+  Use InfluxQL miscellaneous functions to perform different operations in
+  InfluxQL queries.
+menu:
+  influxdb_cloud_dedicated:
+    name: Miscellaneous
+    identifier: influxql-misc-functions
+    parent: influxql-functions
+weight: 206
+---
+
+Use InfluxQL miscellaneous functions to perform different operations in
+InfluxQL queries.
+
+- [fill()](#fill)
+
+## fill()
+
+Fills _null_ field values returned from empty time windows in `GROUP BY time()`
+queries with a specified fill value.
+
+_Supported only in the [`GROUP BY` clause](/influxdb/cloud-dedicated/reference/influxql/group-by/)._
+
+```sql
+fill(behavior)
+```
+
+#### Arguments
+
+- **behavior**: Defines the behavior of the fill operation. Default is `null`.
```

Comment:
```suggestion
- **behavior**: Defines the behavior of the fill operation. The default behavior when the `FILL` clause is omitted is `FILL(null)`.
```

That might be interpreted as omitting the value, so that `fill()` is accepted.